src/test/shell/bazel/remote/build_without_the_bytes_test.sh

```diff
@@ -545,6 +545,34 @@ EOF
   expect_log "uri:.*bytestream://example.com/my-instance-name/blobs"
 }
 
+function test_undeclared_test_outputs_bep() {
+  # Test that when using --remote_download_minimal, undeclared outputs in a test
+  # are reported by BEP
+  mkdir -p a
+  cat > a/BUILD <<EOF
+sh_test(
+  name = "foo",
+  srcs = ["foo.sh"],
+)
+EOF
+  cat > a/foo.sh <<'EOF'
+echo foo
+echo bar > $TEST_UNDECLARED_OUTPUTS_DIR/bar.txt
```

Comment:
Nit: this can be simply `touch $TEST_UNDECLARED_OUTPUTS_DIR/foo.txt`, since we're not verifying the test stdout or the file contents anywhere?

Reply:
Done.